internal/mode/static/nginx/config/variable_names_test.go

```diff
@@ -24,19 +24,13 @@ func TestConvertStringToSafeVariableName(t *testing.T) {
 		},
 	}
 	for _, test := range tests {
-		if result := convertStringToSafeVariableName(test.s); result != test.expected {
-			t.Errorf(
-				"convertStringToSafeVariableName() mismatch for test %q; expected %s, got %s",
-				test.msg,
-				test.expected,
-				result,
-			)
-		}
+		g := NewWithT(t)
+		g.Expect(convertStringToSafeVariableName(test.s)).To(Equal(test.expected))
 	}
 }
 
 func TestGenerateAddHeaderMapVariableName(t *testing.T) {
-	g := NewGomegaWithT(t)
+	g := NewWithT(t)
```

Comment:
Same here with subtests. There's probably more that can be changed to subtests but I'll stop leaving comments.